main.tf

```diff
@@ -61,12 +61,17 @@ resource "azurerm_function_app" "function_app" {
     "FUNCTIONS_WORKER_RUNTIME"              = "python"
     "APPINSIGHTS_INSTRUMENTATIONKEY"        = azurerm_application_insights.appinsight.instrumentation_key
     "APPLICATIONINSIGHTS_CONNECTION_STRING" = "InstrumentationKey=${azurerm_application_insights.appinsight.instrumentation_key};IngestionEndpoint=https://${var.location}-0.in.applicationinsights.azure.com/"
-    "USER_ASSIGNED_IDENTITY_APP_ID"         = data.azurerm_user_assigned_identity.identity.client_id,
-    "LOKI_USERNAME"                         = var.loki_authentication.username,
-    "LOKI_PASSWORD"                         = var.loki_authentication.password,
-    "LOKI_ENDPOINT"                         = var.loki_endpoint_url,
-    "LOKI_LABEL_NAMES"                      = var.loki_label_names,
-    "RESOURCE_GRAPH_QUERY_IDS"               = var.resource_graph_query_ids,
+    "USER_ASSIGNED_IDENTITY_APP_ID"         = data.azurerm_user_assigned_identity.identity.client_id
+    "LOKI_USERNAME"                         = var.loki_authentication.username
+    "LOKI_PASSWORD"                         = var.loki_authentication.password
+    "LOKI_ENDPOINT"                         = var.loki_endpoint_url
+    "LOKI_LABEL_NAMES"                      = var.loki_label_names
+    "RESOURCE_GRAPH_QUERY_IDS"              = var.resource_graph_query_ids
+    "STORAGE_ACCOUNT_CONNECTION"            = azurerm_storage_account.sa.primary_connection_string
+    "CONTAINER_NAME"                        = azurerm_storage_container.sacontainer.name
+    "ENABLE_LOKI_PUBLISHER" : var.enable_loki_publisher
+    "ENABLE_AZURE_BLOB_PUBLISHER" : var.enable_azure_blob_publisher
```

Comment:
will this work with `:` instead `=`?

Reply:
Wow, indeed this is a typo although I've deployed this version yesterday in the dev subscription and it's running ok :smile: . Nevertheless, I'm changing it right now, thanks for the heads-up